addon/templates/components/frost-button.hbs

```diff
@@ -1,3 +1,4 @@
+{{! Template for the frost-button component }}
```

Comment:
I'd prefer to rely on editors to handle distinguishing file names, Atom / VSC / WebStorm / Sublime are all capable of handling this